qemu/tests/rh_qemu_update.py

```diff
@@ -0,0 +1,228 @@
+from avocado.utils import process, distro
+from distutils.version import LooseVersion
+from virttest import error_context
+
+QUERY_TIMEOUT = 360
+INSTALL_TIMEOUT = 360
+OPERATION_TIMEOUT = 1200
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test update of qemu-kvm
+    1) Boot the VM
+    2) Install qemu-kvm metapackage from compose
+    3) Clone component_management tool
+    4) Verify available, host and guest qemu versions
+    5) Update qemu-kvm using component_management
+    6) Verify installed qemu-kvm version
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    qemu_package = params['qemu_package_install']
+
+    def install_qemu(session):
+        """
+        Install compose version of qemu-kvm pkg by the name provided in cfg
+        """
+        cmd = "{} install -y {}".format(
+            params['pm_tool'], qemu_package
```

Comment:
params['pm_tool'] be used for 6 times, I would like to suggest handle with the same way as qemu_package